src/applications/beta-enrollment/containers/BetaEnrollmentButton.jsx

```diff
@@ -70,7 +71,7 @@ class BetaEnrollmentButton extends React.Component {
       );
     }
 
-    let buttonText = 'Turn On Beta Tools';
+    let buttonText = this.props.buttonText || 'Turn On Beta Tools';
```

Comment:
Out of scope for this PR, but `Turn On Beta Tools` is maybe misleading, isn't it? Since this button will only enable a single beta tool. :man_shrugging: 

I like the addition of the `buttonText`. What do you think about making `Turn On Beta Tools` the default value for `buttonText`?

Reply:
Agree with default prop